lib/classes/CLI.js

```diff
@@ -65,8 +65,14 @@ class CLI {
 
   generateMainHelp() {
     this.consoleLog('');
+    this.consoleLog(chalk.yellow('The Serverless Framework - https://serverless.com'));
```

Comment:
Should we use the ASCI Art here? Imho help is the right place to do it and would add something nice visually I think.